Flaky test: test_overcommit_tracker

_Important: This issue was automatically generated and is used by CI for matching failures. DO NOT modify the body content. DO NOT remove labels._

Test name: test_overcommit_tracker
Failure reason: AssertionError: overcommitted query is not killed (attempt 3: overcommitted_killed=False, finished=True)
CI report: [Integration tests (amd_llvm_coverage, 2/5)](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=94690&sha=298cf42e6cd833562f76eef191bfd491b6937198&name_0=PR&name_1=Integration%20tests%20%28amd_llvm_coverage%2C%202%2F5%29)

Failing test history: [cidb](https://play.clickhouse.com#V0lUSAogICAgOTAgQVMgaW50ZXJ2YWxfZGF5cwpTRUxFQ1QKICAgIHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBBUyBkYXksCiAgICBjb3VudCgpIEFTIGZhaWx1cmVzLAogICAgZ3JvdXBVbmlxQXJyYXkocHVsbF9yZXF1ZXN0X251bWJlcikgQVMgcHJzLAogICAgYW55KHJlcG9ydF91cmwpIEFTIHJlcG9ydF91cmwKRlJPTSBjaGVja3MKV0hFUkUgKG5vdygpIC0gdG9JbnRlcnZhbERheShpbnRlcnZhbF9kYXlzKSkgPD0gY2hlY2tfc3RhcnRfdGltZQogICAgQU5EIHRlc3RfbmFtZSA9ICd0ZXN0X292ZXJjb21taXRfdHJhY2tlci90ZXN0LnB5Ojp0ZXN0X3VzZXJfb3ZlcmNvbW1pdCcKICAgIC0tIEFORCBjaGVja19uYW1lID0gJ0ludGVncmF0aW9uIHRlc3RzIChhbWRfbGx2bV9jb3ZlcmFnZSwgMi81KScKICAgIEFORCB0ZXN0X3N0YXR1cyBJTiAoJ0ZBSUwnLCAnRVJST1InKQogICAgQU5EIChwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCBPUiBiYXNlX3JlZiBJTiAoJ21hc3RlcicpKQogICAgQU5EIHRlc3RfY29udGV4dF9yYXcgTElLRSAnJUFzc2VydGlvbkVycm9yOiBvdmVyY29tbWl0dGVkIHF1ZXJ5IGlzIG5vdCBraWxsZWQgKGF0dGVtcHQgMzogb3ZlcmNvbW1pdHRlZF9raWxsZWQ9RmFsc2UsIGZpbmlzaGVkPVRydWUpJScKR1JPVVAgQlkgZGF5Ck9SREVSIEJZIGRheSBERVNDCg==)

Test output:
```
File: test_overcommit_tracker/test.py:75 - in test_user_overcommit
    assert overcommitted_killed, f"overcommitted query is not killed ({last_error})"
E   AssertionError: overcommitted query is not killed (attempt 3: overcommitted_killed=False, finished=True)
E   assert False
```

--- a/tests/integration/test_overcommit_tracker/configs/config.d/config.xml
+++ b/tests/integration/test_overcommit_tracker/configs/config.d/config.xml
@@ -1,4 +1,0 @@
-<clickhouse>
-    <!-- Keep the number of threads consistent across queries in the test -->
-    <concurrent_threads_soft_limit_ratio_to_cores>0</concurrent_threads_soft_limit_ratio_to_cores>
-</clickhouse>

--- a/tests/integration/test_overcommit_tracker/test.py
+++ b/tests/integration/test_overcommit_tracker/test.py
@@ -6,7 +6,6 @@ cluster = ClickHouseCluster(__file__)
 
 node = cluster.add_instance(
     "node",
-    main_configs=["configs/config.d/config.xml"],
     user_configs=[
         "configs/users.d/users.xml",
     ],
@@ -22,8 +21,8 @@ def start_cluster():
         cluster.shutdown()
 
 
-USER_TEST_QUERY_A = "SELECT groupArray(number) FROM numbers(2500000) SETTINGS max_memory_usage_for_user=2000000000,memory_overcommit_ratio_denominator=1"
-USER_TEST_QUERY_B = "SELECT groupArray(number) FROM numbers(2500000) SETTINGS max_memory_usage_for_user=2000000000,memory_overcommit_ratio_denominator=80000000"
+USER_TEST_QUERY_A = "SELECT groupArray(number) FROM numbers(2500000) SETTINGS max_threads=1, max_memory_usage_for_user=2000000000, memory_overcommit_ratio_denominator=1"
+USER_TEST_QUERY_B = "SELECT groupArray(number) FROM numbers(2500000) SETTINGS max_threads=1, max_memory_usage_for_user=2000000000, memory_overcommit_ratio_denominator=80000000"
 
 
 def test_user_overcommit():

